### DISCUSSION TITLE
Button to wake the display

### DISCUSSION THREAD

**Discussion by MartinJSa** [cmcgerty/Sofar2mqtt#48] [Ideas]
Hi,

My OLED display is already a bit knackered, having run constantly for around 6 months now. The pixels which are always on are quite dim, cheap display! Would it be an idea to have an optional button to wake the display? Even with nice displays that don't burn out so readily, it would be good to cut a little power and heat from the unit?

Best

Martin

**Comment by double1968** [comment]
I had the same problem. I removed the display from the socket
I'll put it back if needed

Bye

**Comment by cmcgerty** [comment]
I’m on my second display. It is an issue.
